modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java

```diff
@@ -0,0 +1,321 @@
+                description = "Defines a simple get operation with no inputs and a complex",
+                operationId = "getWithPayloadResponse",
+                deprecated = true,
+                responses = {
+                        @ApiResponse(
+                                responseCode = "200",
+                                description = "voila!",
+                                content = @Content(
+                                        mediaType = "application/json",
+                                        schema = @Schema(implementation = SampleResponseSchema.class)
+                                )
+                        ),
+                        @ApiResponse(
+                                responseCode = "default",
+                                description = "boo",
+                                content = @Content(
+                                        mediaType = "*/*",
+                                        schema = @Schema(implementation = GenericError.class)
+                                )
+                        )
+                }
+        )
+        @Path("/path")
+        @GET
+        public void simpleGet() {
+        }
+    }
+
+    static class SampleResponseSchema {
+        @Schema(description = "the user id")
+        public String id;
+    }
+
+    static class GenericError {
+        private int code;
+        private String message;
+    }
+
+    @Test(description = "reads an operation with response examples defined")
+    public void testOperationWithResponseExamples() {
+        String openApiYAML = readIntoYaml(GetOperationWithResponseExamples.class);
+        int start = openApiYAML.indexOf("get:");
+        int end = openApiYAML.length() - 1;
+        String extractedYAML = openApiYAML.substring(start, end);
+        String expectedYAML = "get:\n" +
+                "      summary: \"Simple get operation\"\n" +
+                "      description: \"Defines a simple get operation with no inputs and a complex output\"\n" +
+                "      operationId: \"getWithPayloadResponse\"\n" +
+                "      responses:\n" +
+                "        200:\n" +
+                "          description: \"voila!\"\n" +
+                "          content:\n" +
+                "            application/json:\n" +
+                "              schema:\n" +
+                "                type: \"object\"\n" +
+                "                properties:\n" +
+                "                  id:\n" +
+                "                    type: \"string\"\n" +
+                "                    description: \"the user id\"\n" +
+                "              examples:\n" +
+                "                basic:\n" +
+                "                  summary: \"shows a basic example\"\n" +
+                "                  description: \"basic\"\n" +
+                "                  value: \"{\\\"id\\\": 19877734}\"\n" +
```

Comment:
This is not right, the value should be parsed into an object not shown as an escaped string